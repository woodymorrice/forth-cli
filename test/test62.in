."pi" 3.1415 constant . . pi 0 + .
