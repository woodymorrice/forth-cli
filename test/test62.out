stack empty
stack empty
3.1415
